libs/blocks/carousel/carousel.js

```diff
@@ -179,6 +180,46 @@ function setIndicatorMultiplyer(carouselElements, activeSlideIndicator, event) {
   }
 }
 
+function updateAriaLive(ariaLive, slide) {
+  let text = '';
+  slide.querySelectorAll(':scope > :not(.section-metadata')?.forEach((el) => {
+    text += el.textContent.trim();
+  });
+  if (text) {
+    ariaLive.textContent = text;
+  } else {
+    const img = slide.querySelector('img');
+    if (img) ariaLive.textContent = img.getAttribute('alt');
+  }
+}
+
+function setAriaHiddenAndTabIndex(carouselElements, activeSlide) {
+  const { el: carouselBlock, slides } = carouselElements;
+  const active = activeSlide ?? carouselBlock.querySelector('.carousel-slide.active');
+  const activeSlideIndex = slides
+    .findIndex((ele) => active.isSameNode(ele));
+  const IndexOfShowClass = [...carouselBlock.classList].findIndex((ele) => ele.includes('show-'));
```

Comment:
Nit: we usually define constants/variables to start with lower case letters. Same comment for `NoOfVisibleSlides`. Any reason why that would not be ok? I realize this was here before, but it's probably a good time to standardize.